sdk/formrecognizer/Azure.AI.FormRecognizer/MigrationGuide.md

```diff
@@ -610,8 +612,10 @@ foreach (CustomFormSubmodel submodel in model.Submodels)
 Train a custom model with `4.0.x`:
 ```C# Snippet:FormRecognizerSampleBuildModel
 // For this sample, you can use the training documents found in the `trainingFiles` folder.
-// Upload the forms to your storage container and then generate a container SAS URL.
-// For instructions to set up forms for training in an Azure Storage Blob Container, please see:
+// Upload the documents to your storage container and then generate a container SAS URL. Note
+// that a container URI without SAS is accepted only when the container is public or has a
+// managed identity configured.
+// For instructions to set up documents for training in an Azure Storage Blob Container, please see:
 // https://aka.ms/azsdk/formrecognizer/buildtrainingset
```

Comment:
I noticed this link points to https://docs.microsoft.com/azure/applied-ai-services/form-recognizer/build-training-data-set (v2). I think we should redirect it to https://docs.microsoft.com/azure/applied-ai-services/form-recognizer/how-to-guides/build-custom-model-v3 (v3). What do you think?

Reply:
Thanks for the suggestion! We have a new link for v3 now: https://aka.ms/azsdk/formrecognizer/buildcustommodel